samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2195 @@
+{
```

Comment:
<p>Finally, the model is now ready for training. To train the model, the <code>model.fit</code> method is called and is provided the number of epochs for training and the estimated learning rate suggested by <code>lr_find</code> in the previous step:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>